components/html/modules/content-newsletter-text-only/_main-1.html

```diff
@@ -1,49 +1,38 @@
+<section class="section section-labelled">
+	<h2 class="section__heading">Featured Stories</h2>
+		<article class="story story-featured">
+			<h3 class="story__title story__title-featured">This title is for the featured story. Three lines used if necessary or needed.</h3>
+				<p class="story__subtitle story__subtitle-featured">Subhead if needed</p>
+				<p class="story__teaser story__teaser-featured">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vitae lacinia dui, ac lacinia dolor. Maecenas mattis nulla ac purus mattis, sit amet facilisis dolor luctus. Donec luctus convallis arcu. Nullam adipiscing, magna porttitor pellentesque ornare, magna tortor dignissim justo, vel fermentum massa neque sit amet urna.</p>
 			
 				<a href="#" class="button button--hover-light">Read More</a>
-			</section>
-	</article>
+		</article>
 </section>
 		
-<section class="section">
-	<article>
-		<h2 class="section__heading">Other Stories</h2>
-			
-       		<ul class="list">
-        		<li class="list__listItem">
-        		<span class="fa-stack fa-2x"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-newspaper-o fa-stack-1x"></i></span>
-        		<span class="title-list">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</span></li>
-        		
-        		<li class="list__listItem">
-        		<span class="fa-stack fa-2x"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-newspaper-o fa-stack-1x"></i></span>
-        		<span class="title-list">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</span></li>
-        		
-        		<li class="list__listItem">
-        		<span class="fa-stack fa-2x"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-newspaper-o fa-stack-1x"></i></span>
-        		<span class="title-list">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</span></li>
-    		</ul>
-	</article>
+<section class="section" aria-labelledby="section__heading-other">
+	<h2 class="section__heading" id="section__heading-other">Other Stories</h2>		
+		<ul class="section__list section__list-other">
+			<li class="section__item section__item-other story story-other">
+			<a href="#" class="story__link story__link-other">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</a>
+			</li>
+			<li class="section__item section__item-other story story-other">
+			<a href="#" class="story__link story__link-other">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</a>
+			</li>
+			<li class="section__item section__item-other story story-other">
+			<a href="#" class="story__link story__link-other">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</a>
+			</li>
+		</ul>
 </section>
 
-
-<aside class="cta-archive">
-	<h2 class="cta-archive__title">Want to read more?</h2>
-	<p class="cta-archive__description">Check out our archive section</p>
-	<a href="#" class="button button--hover-light btn-center">Read More</a>
+<aside class="cta" aria-labelledby="cta__title">
```

Comment:
Let's add the class `cta-archive` after the `cta` class. This modifier could be used in the future if we want to differentiate this archive cta from another cta that might have a different style.